### ISSUE TITLE
CAA  has a  bug

### ISSUE THREAD

**Issue by hi-glenn** [balliegojr/simple-dns#34]
expect  ` caa.xxx.com.           11111   IN      CAA     128 issuewild "example.org"`

but really get  `caa.xxx.com.           11111   IN      CAA     128 issuewild "\012\"example.org"`

```
        let rec = ResourceRecord::new(
            nm.clone(),
            CLASS::IN,
            11111,
            RData::CAA(CAA {
                flag: 128,
                tag: CharacterString::new(_tag.as_bytes()).unwrap(), 
                value: CharacterString::new(b"\"example.org").unwrap()
            }),
        );

dig @127.0.0.1 caa.xxx.com caa

; <<>> DiG 9.11.4-P2-RedHat-9.11.4-26.P2.el7_9.9 <<>> @127.0.0.1 caa.xxx.com caa
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 36940
;; flags: qr aa rd; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0
;; WARNING: recursion requested but not available

;; QUESTION SECTION:
;caa.xxx.com.                  IN      CAA

;; ANSWER SECTION:
caa.xxx.com.           11111   IN      CAA     128 issuewild "\012\"example.org"

```

**Comment by balliegojr** [comment]
Thanks for reporting this.

I have published a new version (0.8.0), can you verify if the fix works?

**Comment by hi-glenn** [comment]
> Thanks for reporting this.
> 
> I have published a new version (0.8.0), can you verify if the fix works?

solved !  👍

used as the follow
```rust
value: b"example.org".into()
```
